src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -332,7 +332,7 @@ updateUserProfile profile = setProfile profile >> getUserProfile
 
 getWallet :: WalletWebMode ssc m => CAddress -> m CWallet
 getWallet cAddr = do
-    balance <- getBalance =<< decodeCAddressOrFail cAddr
+    balance <- fmap mkCCoin . getBalance =<< decodeCAddressOrFail cAddr
     meta <- getWalletMeta cAddr >>= maybe noWallet pure
     pure $ CWallet cAddr balance meta
   where
```

Comment:
If it works as is, I wonder why Darko asked to pass `Coin` as `String` )  Okay then, let it be so for a while